docs/faq/README.md

```diff
@@ -200,9 +200,7 @@ class MyComponent extends LitElement {
     // ...
   }
   render() {
-    return html`
-      <button @click=${this.myFunction}>click</button>
-    `;
+    return html` <button @click=${this.myFunction}>click</button> `;
```

Comment:
```suggestion
    return html`<button @click=${this.myFunction}>click</button>`;
```

Nit: the way this collapsed the previous white space, there are now space at the beginning and end of all the templates. It's the _same_, but is it what we _want_/what we want to exhibit for others?

Reply:
These will need to be trimmed manually groomed now indeed :)